ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master.yaml

```diff
@@ -0,0 +1,52 @@
+base_images:
+  openshift_release_golang-1.15:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
+images:
+- dockerfile_path: Dockerfile.assisted-installer
+  from: src
+  inputs:
+    openshift_release_golang-1.15:
+      as:
+      - registry.svc.ci.openshift.org/openshift/release:golang-1.15
+  to: assisted-installer
+promotion:
+  name: "4.7"
+  namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.7"
+  namespace: ocp
+# test_binary_build_commands: |
+#   make build-openshift-ci-test-bin
+tests:
+- as: fmt
+  commands: gofmt -l .
+  container:
+    from: test-bin
+- as: fmtcheck
+  commands: test -z $(gofmt -l .)
+  container:
+    from: test-bin
+- as: vet
+  commands: go vet ./...
+  container:
+    from: test-bin
+- as: lint
+  commands: make lint
+  container:
+    from: test-bin
```

Comment:
I'm not sure that we need so many steps. One `lint` test can include them all.

Reply:
My reason for having (what ended up being `format` and `format-check` was that you can use `format` to tell you why `format-check` failed on your PR). I've found it helpful in the past to know that I can dig into why my PR is failing by just looking through the different prow jobs.